packages/redux/src/entities/types/sharedWishlistItem.types.ts

```diff
@@ -0,0 +1,27 @@
+import type { AttributesAdapted, PriceAdapted } from '../../helpers/adapters';
+import type { SharedWishlistItem as OrigianlSharedWhislistItem } from '@farfetch/blackout-client';
+import type { ProductEntity } from './product.types';
+
+export type SharedWishlistItemEntity = {
+  id: OrigianlSharedWhislistItem['id'];
```

Comment:
According to the sharedWishlistItem schema, this is not the correct type of the entity. I think you are missing some code inside the schema to adapt the price and size properties. Also, as the properties are almost the same as from the `SharedWishlistItem` type you can define it like this:

```js 
export type SharedWishlistItemEntity = Omit<SharedWishlistItem, 'price' | 'size'> & {
  size: AttributesAdapted;
  price: PriceAdapted;
}
```

Can you take a look at it?

Reply:
Looks good ideia. Changed, please review.